packages/solarwinds-apm/src/init.ts

```diff
@@ -42,80 +41,78 @@ import {
 } from "@solarwinds-apm/instrumentations"
 import * as sdk from "@solarwinds-apm/sdk"
 
-import packageJson from "../package.json"
-import { type ExtendedSwConfiguration, printError, readConfig } from "./config"
+import { version } from "../package.json"
+import {
+  type ExtendedSwConfiguration,
+  printError,
+  readConfig,
+} from "./config.js"
+import { setter } from "./symbols.js"
 
 export async function init() {
-  const id = `${packageJson.name}@${packageJson.version}`
-  const initSymbol = Symbol.for(`${id}/init`)
-
-  if (!(initSymbol in globalThis)) {
-    Object.defineProperty(globalThis, initSymbol, {
-      value: true,
-      writable: false,
-      enumerable: false,
-      configurable: false,
-    })
+  // init only once
+  const setInit = setter("init")
+  if (!setInit) return
+  setInit()
```

Comment:
Big diff here but it's just an early return and everything being moved out of the if statement and losing an indentation level.